tests/nested/core/core22-basic/task.yaml

```diff
@@ -0,0 +1,32 @@
+summary: Run a smoke test on UC22 with encryption enabled
+
+details: |
+    This test checks basic snapd commands on UC22 with secure boot and encryption enabled
+
+systems: [ubuntu-22.04-64]
+
+execute: |
+    echo "Wait for the system to be seeded first"
+    tests.nested exec "sudo snap wait system seed.loaded"
+
+    echo "Ensure 'snap install' works"
+    # The install command could cause a ssh break, so || true is used
+    # and then we check the installation was completed successfully
+    tests.nested exec "sudo snap install test-snapd-sh" || true
```

Comment:
do you remember how installing an application snap could lead to an ssh break? that seems wrong to me, though I can see why this would be necessary when updating the base/kernel/gadget/snapd snaps

Reply:
I don't remember well but I think it is related to an issue that we used to see initially in uc20 but it not present anymore, I'll try removing it.